tests/test_three_d.py

```diff
@@ -80,8 +104,4 @@ def test_trace_close_to_axis(self):
         # We don't want to make our tests run too long, therefore the meshs size is relatively large
         # therefore we can an accuracy of only 10% here. If the mesh size is decreased the correspondence 
         # is much better.
-        assert np.allclose(intersection, intersection_radial, rtol=10e-2)
-
-
-
-
+        assert np.allclose(intersection, intersection_radial, rtol=10e-2)
```

Comment:
Note that you have made no changes here, but this still shows up as changed in your PR. Some whitespace issues with your editor/git ?